std/math.d

```diff
@@ -6832,6 +6832,149 @@ if (isFloatingPoint!(F) && isFloatingPoint!(G))
     assert(approxEqual(pow(twoI, three), 8.0));
 }
 
+/** Computes the value of a positive integer x, raised to the power n, modulo m.
+ *
+ * All values have to be unsigned types, otherwise it will raise a compilation error.
+ *
+ */
+Unqual!(Largest!(F, H)) powmod(F, G, H)(F x, G n, H m) @nogc @safe pure nothrow
+if (isIntegral!(F) && isIntegral!(G) && isIntegral!(H) &&
+    is (Unsigned!F == F) && is (Unsigned!G == G) && is (Unsigned!H == H))
```

Comment:
The template constraint can be simplified if you use [`isUnsigned`](https://dlang.org/phobos-prerelease/std_traits#isUnsigned) directly. Also when there's a single template argument, the convention is to drop the parentheses.

```D
if (isUnsigned!F && isUnsigned!G && isUnsigned!H)
```